src/views/Community/View.js

```diff
@@ -0,0 +1,141 @@
+import React, { Component } from 'react'
+// import Dashboard from '../Dashboard'
+import { connect } from 'react-redux'
+import { compose } from 'redux'
+import { Row, Progress } from 'reactstrap';
+import { sendCommunityData } from '../../store/actions/community'
+import Payment from '../../components/Payment'
+import { isAuthUserType } from '../../services/Auth'
+import { Link } from 'react-router-dom'
+import Page404 from '../../views/Pages/Page404'
+
+
+class View extends Component {
+  constructor(props) {
+    super(props)
+
+    this.deleting = this.deleting.bind(this)
+  }
+
+  componentDidMount() {
+    let id = this.props.match.params.id * 1;
+    let { sendData, communityData } = this.props
+    if (!isNaN(id)) {
+      if (communityData.communityData.length === 0) {
+        sendData(id, 'get');
+      }
+    }
+  }
+
+  deleting(id) {
+    let formData = new FormData();
+    formData.append('type', 'delete');
+    formData.append('id', id);
+    this.props.sendData(formData,'post');
+    this.props.history.push('/community')
+  }
+
+
+
+  render() {
+    let id = this.props.match.params.id * 1;
+    let { loading, error, errorMessage, communityData } = this.props.communityData
+    let u;
+    let button;
+    switch (isAuthUserType()){
+      case 'customer':
+        button = (id) => { return ( <span className="container"><div className="payment_h2">Card Payment</div>
+         <Payment info={{view:'community',id}} />  <div className="btn_view">
+            <button type="submit" name="button" form="card_payment_form" className="btn_donate btn delete">Donate</button>
+          </div>
+         </span> )};
+        break;
+      case 'admin':
+        button = (id) => { return(<div className="btn_view">
+        <button className="delete btn" onClick={ () => this.deleting(id) }>Delete</button>
+        <Link to={'/community/edit/'+ id}><button className="edit btn">Edit</button></Link>
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Operator '+' must be spaced.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)